ui/app/components/clients/attribution.js

```diff
@@ -4,36 +4,31 @@
  */
 
 import Component from '@glimmer/component';
-import { parseAPITimestamp } from 'core/utils/date-formatters';
-import { isSameMonth } from 'date-fns';
 
 /**
  * @module Attribution
- * Attribution components display the top 10 total client counts for namespaces or auth methods (mounts) during a billing period.
+ * Attribution components display the top 10 total client counts for namespaces or mounts during a billing period.
  * A horizontal bar chart shows on the right, with the top namespace/auth method and respective client totals on the left.
  *
  * @example
  *  <Clients::Attribution
- *    @totalClientAttribution={{this.totalClientAttribution}}
- *    @newClientAttribution={{this.newClientAttribution}}
- *    @selectedNamespace={{this.selectedNamespace}}
- *    @startTimestamp={{this.startTime}}
- *    @endTimestamp={{this.endTime}}
- *    @isHistoricalMonth={{false}}
- *    @responseTimestamp={{this.responseTimestamp}}
+ *    @noun="auth mount"
+ *    @attribution={{array (hash label="ns1" clients=100)}}
+ *    @responseTimestamp="2018-04-03T14:15:30"
+ *    @isSecretsSyncActivated={{true}}
  *  />
  *
- * @param {array} totalClientAttribution - array of objects containing a label and breakdown of client counts for total clients
- * @param {array} newClientAttribution - array of objects containing a label and breakdown of client counts for new clients
- * @param {string} selectedNamespace - namespace selected from filter bar
- * @param {string} startTimestamp - timestamp string from activity response to render start date for CSV modal and whether copy reads 'month' or 'date range'
- * @param {string} endTimestamp - timestamp string from activity response to render end date for CSV modal and whether copy reads 'month' or 'date range'
+ * @param {string} noun - noun which reflects the type of data and used in title. Should be "namespace" (default) or "auth mount"
```

Comment:
per slack conversation 
```suggestion
 * @param {string} noun - noun which reflects the type of data and used in title. Should be "namespace" (default) or "mount"
```